woocommerce/woocommerce
# Remote Specs Testing package: Add new package

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Related to https://github.com/woocommerce/woocommerce/issues/44262 and https://github.com/woocommerce/woocommerce/issues/44417

This PR adds scaffolds out the beginning of a PHP package so work can begin on the two above issues at the same time. 

pnpm doesn't work with `composer.json`, so we must have a `package.json` in order to make use of pnpm's commands and filtering. This way there is uniform commands for things like adding a changelog. Unfortunately, this means the name must be `@woocommerce/remote-inbox-notification-testing` in `package.json` and `woocommerce/remote-inbox-notification-testing` in `composer.json`. This distinction is, however, already present in our JS packages.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure `pnpm install` works.
2. Make sure pnpm filtering works as well as the changelogger: `pnpm --filter="@woocommerce/remote-inbox-notification-testing" changelog add`.
3. Make sure added files look correct.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
